cmd/cmdtest/testutils.go

```diff
@@ -397,3 +398,21 @@ func GetEnvStacksList() string {
 	}
 	return stacksList
 }
+
+func cleanUpTestDepDockerVolumes(t *testing.T) []string {
+	var testDepVolumes []string
+	output, err := RunCmdExec("docker", []string{"volume", "ls", "--format", "{{.Name}}"}, t)
+	if err != nil {
+		t.Logf("Error listing docker volumes: %v", err)
+	}
+	volumes := strings.Fields(output)
+	for _, volume := range volumes {
```

Comment:
Instead of looping through all of the volumes you can specifically target `t.ProjectName`. That way we wouldn't need to `ls` either - I am not sure if you'd need to verify if the volume exists before attempting to delete it, though; if you execute `docker volume rm <invalid>` it would throw a `No such volume`, but you could probably just print that and move on.